test/integration/config-ci.js

```diff
@@ -0,0 +1,16 @@
+// this file is for use in CircleCI continuous integration environment
+module.exports = {
+    seleniumServerURL: {
+        hostname    : 'ondemand.saucelabs.com',
+        port        : 80,
+    },
+    driverCapabilities: {
+        platform            : 'Windows 7',
+        'tunnel-identifier' : 'circle-' + process.env.CIRCLE_PROJECT_USERNAME + '-' + process.env.CIRCLE_PROJECT_REPONAME + '-' + process.env.CIRCLE_BUILD_NUM + '-' + process.env.CIRCLE_NODE_INDEX
+    },
+    tags        : [ 'circle-ci', '#' + process.env.CIRCLE_BUILD_NUM ],
+    views       : [ 'Verbose', 'SauceLabs' ],
```

Comment:
This is currently ignored because the original `config.json` also defines views, which in turn prevents the SauceLabs from being active, thus preventing build status to be sent to SauceLabs 😞

Two possible fixes:

1. Remove `views` from the original `config.json`.
2. Enforce `views` through the `--config` CLI switch in `circle.yml`.